File: public/phase-2-20160620022203/JAVASCRIPT/mssum1.js
// Name : MSSUM1
// input:
// n :  the number of elements in the array
// q :  the number of queries correspondingly
// a :  the array elements
// lr : each row corresponds to l and r
var index,sum_of_replies;
var elements = readline().split(" ");
var n = parseInt(elements[0], 10);
var q = parseInt(elements[1], 10);
var a = [];
var a_elements = readline().split(' ');
for(index=0;index<n;index++)
	a.push(parseInt(a_elements[index], 10));
var lr = new Array([]);
var lr_elements = "";
for(idx=0;idx<q;idx++){
	lr_elements = readline().split(' ');
	lr[idx] = new Array([]);
	for(jdx=0;jdx<2;jdx++){
		lr[idx][jdx] = parseInt(lr_elements[jdx], 10);
	}
}


// write your code here
// store your results in `sum_of_replies`

// Dummy Data
sum_of_replies = 25;


// output
print(sum_of_replies);

File: public/phase-2-20160620022203/JAVA/mssum1.java
// Name : MSSUM1
import java.util.*;
import java.lang.*;
import java.io.*;
class Main {
	public static void main (String[] args) throws java.lang.Exception {
		// input:
		// n :  the number of elements in the array
		// q :  the number of queries correspondingly
		// a :  the array elements
		// lr : each row corresponds to l and r
		int index;
		int idx;
		int jdx;
		Scanner in = new Scanner(System.in);
		String[] elements = (in.nextLine()).split(" ");
		int n = Integer.parseInt(elements[0]);
		int q = Integer.parseInt(elements[1]);
		int[] a = new int[n];
		String[] a_elements = (in.nextLine()).split(" ");
		for(index=0;index<n;index++)
			a[index] = Integer.parseInt(a_elements[index]);
		int[][] lr = new int[q][2];
		String[] lr_elements = new String[100];
		for(idx=0;idx<q;idx++){
			lr_elements = (in.nextLine()).split(" ");
			for(jdx=0;jdx<2;jdx++){
				lr[idx][jdx] = Integer.parseInt(lr_elements[jdx]);
			}
		}


		// write your code here
		// store your results in `sum_of_replies`

		// output
		// Dummy Data
		int sum_of_replies=25;

		System.out.println(sum_of_replies);

	}
}